fictadvisor-api/src/v2/database/repositories/QuestionRepository.ts

```diff
@@ -12,14 +13,11 @@ export class QuestionRepository {
     questionRoles: true,
   };
 
-  async findMany (data: Prisma.QuestionFindManyArgs) {
+  async findMany (data: Prisma.QuestionFindManyArgs): Promise<DbQuestionWithRoles[]> {
     return this.prisma.question.findMany({
       include: this.include,
       ...data,
-      orderBy: {
-        order: 'asc',
-      },
-    });
+    }) as unknown as Promise<DbQuestionWithRoles[]>;
```

Comment:
Remove `: Promise<DbQuestionWithRoles[]>` and `as unknown as Promise<DbQuestionWithRoles[]>`